docker-compose-mm.yml

```diff
@@ -0,0 +1,92 @@
+version: '3'
```

Comment:
Really good to keep this version with Scout connected to MatchMaker, we can reuse it for sure when we need to test changes that have an effect on the interaction of the 2 repositories. What I'd do is renaming into something more self-explanatory, like `docker-compose-matchmaker.yml ` and move it where we keep other containers. What about creating a new directory under scout/containers/development (where there are already kubernetes and systemd) and move it there?